### DISCUSSION TITLE
error: push_call_printf: stack size is going past 100.

### DISCUSSION THREAD

**Discussion by amelsen** [scandum/tintin#125] [Bugs]
![image](https://user-images.githubusercontent.com/3955258/191932310-1a74458b-614c-482b-a972-5d664b440359.png)

Suddenly my client starts giving me his error? What can cause this... it's a very very simple #action that triggers it.. absolutely nothing complex about it?

**Comment by scandum** [comment]
It's typically caused by an infinite loop. Either an alias calling itself, or an action using #showme that triggers itself.
